openshift/release/kn.yaml

```diff
@@ -1,4 +1,24 @@
 plugins:
+  - name: kn-plugin-source-kafka
```

Comment:
just out of curiosity, what is this kn.yaml file used for ?

Reply:
It's config for `knb` to populate `plugin_register.go`, add dependency to go.mod etc.